datasets/klue/klue.py

```diff
@@ -0,0 +1,545 @@
+            proposed for this task, which motivates us to construct the first Korean topic
+            classification benchmark. In this task, given a news headline, a text classifier must
+            predict a topic which is one of politics, economy, society, culture, world, IT/science,
+            and sports. Macro-F1 score is used to evaluate a system."""
+            ),
+            data_url=_DATA_URLs["ynat"],
+            url=_DESCRIPTION_URLs["ynat"],
+            file_map={
+                "train": "ynat-v1_train.json",
+                "dev": "ynat-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="sts",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "labels": {
+                    "label": datasets.Value("float64"),
+                    "real-label": datasets.Value("float64"),
+                    "binary-label": datasets.ClassLabel(names=["negative", "positive"]),
+                },
+                "annotations": {
+                    "agreement": datasets.Value("string"),
+                    "annotators": [datasets.Value("string")],
+                    "annotations": [datasets.Value("int32")],
+                },
+            },
+            description=textwrap.dedent(
+                """\
+            STS is a task which aims to predict the semantic similarity of two input sentences as
+            a real value between 0 and 5. Note that we furthure binarized the prediction scores
+            into two classes with a threshold score 3.0 (paraphrased or not) and evaluated with
+            a classification metric.
+            """
+            ),
+            data_url=_DATA_URLs["sts"],
+            url=_DESCRIPTION_URLs["sts"],
+            file_map={
+                "train": "klue-sts-v1_train.json",
+                "dev": "klue-sts-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="nli",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "premise": datasets.Value("string"),
+                "hypothesis": datasets.Value("string"),
+                "gold_label": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
+                "author": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
+                "label2": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
+                "label3": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
+                "label4": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
+                "label5": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
```

Comment:
```suggestion
                "guid": datasets.Value("string"),
                "source": datasets.Value("string"),
                "premise": datasets.Value("string"),
                "hypothesis": datasets.Value("string"),
                "label": datasets.ClassLabel(names=["entailment", "contradiction", "neutral"]),
```
I think they don't need to be added because they just show how curators decide the gold label and they can give confusion to users.